db/migrate/20210730052719_add_columns_to_users.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddColumnsToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :postal_code, :string
+    add_column :users, :address, :string
+    add_column :users, :self_introduction, :text
```

Comment:
`change_table` を使って同じような migration を書けないでしょうか。

```suggestion
    change_table :users, do |t|
      t.string :postal_code
      t.string :address
      t.text :self_introduction
    end
```

Reply:
書けると思います。何か気になることがありますか？

Reply:
`change_table` を使ったほうが `:users` の重複をなくせるのではないかと。

Reply:
なるほど、たしかに重複がなくなって良いですね！理解しました、ありがとうございます。

Reply:
（参考）migrationファイルは保守する必要がないファイルなので、多少の重複があってもあまり気にならない人です。